src/components/recovery/RecoveryDescription/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { Typography } from '@mui/material'
+import { useMemo } from 'react'
+import type { ReactElement } from 'react'
+
+import EthHashInfo from '@/components/common/EthHashInfo'
+import { InfoDetails } from '@/components/transactions/InfoDetails'
+import ErrorMessage from '@/components/tx/ErrorMessage'
+import { useIsGuardian } from '@/hooks/useIsGuardian'
+import useSafeInfo from '@/hooks/useSafeInfo'
+import { logError, Errors } from '@/services/exceptions'
+import { getRecoveredSafeInfo } from '@/services/recovery/transaction-list'
+import type { RecoveryQueueItem } from '@/services/recovery/recovery-state'
+
+export function RecoveryDescription({ item }: { item: RecoveryQueueItem }): ReactElement {
+  const { args, isMalicious } = item
+  const { safe } = useSafeInfo()
+  const isGuardian = useIsGuardian()
```

Comment:
I thought Guardians were renamed to Recoverers across the app and code. Was that reverted?

Reply:
It was. This was a merge issue. I've fixed it in 0851e3ee0ea6d138580a1fd0437fe7c7b59a6e64.